shaders/mobile_nerf/mlp_combo.frag

```diff
@@ -0,0 +1,238 @@
+               BIAS_0_COUNT + BIAS_1_COUNT + BIAS_2_COUNT)/4];        // Array of floats
+} weights_arr[];
+
+vec3 evaluateNetwork(  vec4 f0,  vec4 f1,  vec4 viewdir,  uint idx) {
+
+        vec3 res;
+
+        int bias_0_ind = WEIGHTS_0_COUNT + WEIGHTS_1_COUNT + WEIGHTS_2_COUNT;
+        vec4 intermediate_one[4] = vec4[](
+           weights_arr[nonuniformEXT(idx)].data[bias_0_ind/4],
+           weights_arr[nonuniformEXT(idx)].data[bias_0_ind/4 + 1],
+           weights_arr[nonuniformEXT(idx)].data[bias_0_ind/4 + 2],
+           weights_arr[nonuniformEXT(idx)].data[bias_0_ind/4 + 3]
+        );
+
+        #if 0
+        for (int j = 0; j < 11; ++j) {
+            float input_value = 0.0;
+            if (j < 4) {
+            input_value =
+                (j == 0) ? f0.r : (
+                (j == 1) ? f0.g : (
+                (j == 2) ? f0.b : f0.a));
+            } else if (j < 8) {
+            input_value =
+                (j == 4) ? f1.r : (
+                (j == 5) ? f1.g : (
+                (j == 6) ? f1.b : f1.a));
+            } else {
+            input_value =
+                (j == 8) ? viewdir.r : (
+                (j == 9) ? -viewdir.b : viewdir.g); //switch y-z axes
+            }
+            for (int i = 0; i < 16; ++i) {
+            intermediate_one[i] += input_value * weights0[ j*16 + i];
+                //texelFetch(weightsZero, ivec2(j, i), 0).x;
+            }
+        }
+        #endif
+
+#define  APPLY_WEIGHTS_0(multiplier, weightFirstInd) \
+        intermediate_one[ 0] += (multiplier) * weights_arr[nonuniformEXT(idx)].data[ weightFirstInd/4]; \
+        intermediate_one[ 1] += (multiplier) * weights_arr[nonuniformEXT(idx)].data[ weightFirstInd/4 + 1]; \
+        intermediate_one[ 2] += (multiplier) * weights_arr[nonuniformEXT(idx)].data[ weightFirstInd/4 + 2]; \
+        intermediate_one[ 3] += (multiplier) * weights_arr[nonuniformEXT(idx)].data[ weightFirstInd/4 + 3];
+
+         APPLY_WEIGHTS_0( f0.r,         0)
+         APPLY_WEIGHTS_0( f0.g,        16)
+         APPLY_WEIGHTS_0( f0.b,        32)
+         APPLY_WEIGHTS_0( f0.a,        48)
+         APPLY_WEIGHTS_0( f1.r,        64)
+         APPLY_WEIGHTS_0( f1.g,        80)
+         APPLY_WEIGHTS_0( f1.b,        96)
+         APPLY_WEIGHTS_0( f1.a,       112)
+         // For models form original mobile nerf, use the original code
+         APPLY_WEIGHTS_0( viewdir.r,  128)
+         APPLY_WEIGHTS_0( -viewdir.b, 144)
+         APPLY_WEIGHTS_0( viewdir.g,  160)
+
+        int bias_1_ind = WEIGHTS_0_COUNT + WEIGHTS_1_COUNT + WEIGHTS_2_COUNT +
+                         BIAS_0_COUNT;
+        vec4 intermediate_two[4] = vec4[](
+           weights_arr[nonuniformEXT(idx)].data[bias_1_ind/4],
+           weights_arr[nonuniformEXT(idx)].data[bias_1_ind/4 + 1],
+           weights_arr[nonuniformEXT(idx)].data[bias_1_ind/4 + 2],
+           weights_arr[nonuniformEXT(idx)].data[bias_1_ind/4 + 3]
+        );
+
+         #if 0
```

Comment:
As above.

Reply:
Removed these and all other occurrences.